ucx_perftest building is BROKEN

commit 6777d78e2923c3dccd73942e48a02190960c4aab
```
$ ./configure --disable-static && make V=1
[...]
# CC      perftest.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I../../.. -O3 -g -Wall -Werror -msse4.1 -mavx -I/home/abuild/rpmbuild/BUILD/ucx/src -I/home/abuild/rpmbuild/BUILD/ucx/src ./perftest.c -c  -fPIC -DPIC -o .libs/perftest.o
# CCLD    libucxperf.la
libtool: link: gcc perftest.o -o .libs/ucx_perftest  ./.libs/libucxperf.a /home/abuild/rpmbuild/BUILD/ucx/src/ucp/.libs/libucp.so /home/abuild/rpmbuild/BUILD/ucx/src/uct/.libs/libuct.so /home/abuild/rpmbuild/BUILD/ucx/src/ucs/.libs/libucs.so -lm -lpthread -lrt -ldl -Wl,-rpath -Wl,/usr/lib64
gcc: error: perftest.o: No such file or directory
Makefile:894: recipe for target 'ucx_perftest' failed
```

--- a/src/tools/perf/Makefile.am
+++ b/src/tools/perf/Makefile.am
@@ -57,16 +57,16 @@ ucx_perftest_SOURCE_FILES = \
 #
 # Add custom rules to compile and link ucx_perftest with MPICC
 #
-perftest.o: $(ucx_perftest_SOURCE_FILES)
+perftest.lo: $(ucx_perftest_SOURCE_FILES)
 	$(AM_V_CC)
 	@$(LIBTOOL) $(AM_V_lt) --mode=compile --tag=CC $(ucx_perftest_CC) \
 		$(DEFS) -I$(top_builddir) $(CFLAGS) $(AM_CPPFLAGS) $(ucx_perftest_CPPFLAGS) \
 		$(ucx_perftest_CFLAGS) $(ucx_perftest_SOURCE_FILES) -c -o $@
 
-ucx_perftest$(EXEEXT): perftest.o $(ucx_perftest_LDADD)
+ucx_perftest$(EXEEXT): perftest.lo $(ucx_perftest_LDADD)
 	$(AM_V_CCLD)
 	@$(LIBTOOL) $(AM_V_lt) --mode=link --tag=CC $(ucx_perftest_CC) \
-		perftest.o $(LDFLAGS) $(ucx_perftest_LDFLAGS) $(ucx_perftest_LDADD) -o $@
+		perftest.lo $(LDFLAGS) $(ucx_perftest_LDFLAGS) $(ucx_perftest_LDADD) -o $@
 
 if HAVE_MPIRUN
 .PHONY: ucx test help

